src/pool.c

```diff
@@ -1143,6 +1143,7 @@ hsk_peer_send(hsk_peer_t *peer, const hsk_msg_t *msg) {
 
 static int
 hsk_peer_send_version(hsk_peer_t *peer) {
+  hsk_peer_log(peer, "sending version\n");
```

Comment:
Nit: maybe we use `hsk_debug_log`?

Reply:
I think `hsk_peer_log()` is correct, all the other p2p message logs in this file use it, and it is peer specific, eg:

```
peer 4 (172.104.214.189:44806): connected
peer 4 (172.104.214.189:44806): sending version
peer 4 (172.104.214.189:44806): received verack
peer 4 (172.104.214.189:44806): received version: /hsd:2.0.2/ (28839)
peer 4 (172.104.214.189:44806): sending verack
peer 4 (172.104.214.189:44806): sending sendheaders
peer 4 (172.104.214.189:44806): sending getheaders
peer 4 (172.104.214.189:44806): received 1 addrs
```